pi-manage

```diff
@@ -159,7 +165,7 @@ def create_crl(ca, force=False):
 @ca_manager.option('name', help='The name of the new CA')
 @ca_manager.option('-t', '--type', help='The type of the new CA. The default '
                                         'is "local"')
-def create(name, type):
+def create(name, catype='local'):
```

Comment:
I think renaming ``type`` to ``catype`` is a good idea to avoid shadowing the ``type`` builtin, but unfortunately it breaks the command because flask-script tries to pass a ``type`` argument :-/

```shell
$ ./pi-manage ca create foo                                                                                                                                                              
   
Traceback (most recent call last):
  File "./pi-manage", line 1200, in <module>
    manager.run()
  File "/home/fred/privacyidea/privacyidea/venv/lib/python2.7/site-packages/flask_script/__init__.py", line 417, in run
    result = self.handle(argv[0], argv[1:])
  File "/home/fred/privacyidea/privacyidea/venv/lib/python2.7/site-packages/flask_script/__init__.py", line 386, in handle
    res = handle(*args, **config)
  File "/home/fred/privacyidea/privacyidea/venv/lib/python2.7/site-packages/flask_script/commands.py", line 216, in __call__
    return self.run(*args, **kwargs)
TypeError: <flask_script.commands.Command object at 0x7f6c1569c550>: create() got an unexpected keyword argument 'type'
```

But I think it might be possible to just add ``dest="catype"`` to the @ca_manager.option decorator?

Reply:
That is why we need tests :-D

Reply:
fixed in b065a00f